packages/pharos-site/src/components/Sidenav.tsx

```diff
@@ -76,6 +76,9 @@ const Sidenav: FC = () => {
           >
             Help
           </PharosSidenavLink>
+          <PharosSidenavMenu label="Contributing" expanded={isExpanded('contributing')}>
+            {['Contributing documentation'].map(createSidenavLink.bind(this, 'contributing'))}
```

Comment:
I kinda don't like how on the sidenav the title takes up two lines. Thoughts?

Reply:
Think the title for the item can just be `Documentation`